components/PersonView/PersonDetails.jsx

```diff
@@ -62,10 +62,12 @@ const PersonDetails = ({
               <div className="govuk-summary-list__row">
                 <dt className="govuk-summary-list__key">Address</dt>
                 <dd className="govuk-summary-list__value">
-                  {addressList &&
-                    Object.values(addressList[0]).map((addressLine) => (
-                      <p>{addressLine}</p>
-                    ))}
+                  <p>
+                    {addressList &&
+                      addressList.filter(
```

Comment:
[Picky mode ON]
In this case we can use `find` instead of filter, as there will be only one and the loop is interrupting when it found it 😅 
[Picky mode OFF]